scripts/tasks/size-audit.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path');
+const fs = require('fs');
+
+exports.generateSizeData = function() {
+  const projectName = path.basename(process.cwd());
+  const sizeFilePath = path.join(process.cwd(), 'dist', `${projectName}.json`);

```

Comment:
This should be named according to the purpose of the file being generated. Something like `sizedata.json` - having a project name in the json is fine too like `${projectName}-sizedata.json`